nucypher/characters/control/specifications/bob.py

```diff
@@ -68,6 +51,10 @@ class Retrieve(BaseSchema):
         click=options.option_message_kit(required=False)
     )
 
+    treasure_map = EncryptedTreasureMap(required=True,
+                                        load_only=True,
+                                        click=options.option_treasure_map)
```

Comment:
Since this parameter is now required, the click option will also need to be required. You'll probably have to define the option as a function instead in options.py so that `required=True` can be specified as a parameter.

Reply:
✔️

Reply:
Yes, that's correct.